polaris-react/src/components/AlphaStack/AlphaStack.scss

```
@@ -2,7 +2,7 @@
 
 .AlphaStack {
   // stylelint-disable -- Polaris component custom properties
-  --pc-stack-gap-xs: var(--p-space-4);
+  --pc-stack-gap-xs: none;
```

`gap: none` isn't valid CSS 
```suggestion
  --pc-stack-gap-xs: 0;
```